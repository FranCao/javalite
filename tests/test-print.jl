int i = 3;
int j = 9;
print(i + j);
print(i - j);
print(i / j);

string s = "Foo";
print(s);

double a = 3.14159267;
double b = -2.71828;
print(a + b);
print(a == b);

print(true && false);
print(!false);
print("hello" + " world");
print(1 + 2 * 3 + 4);

string[] strArr = ["hello", "world"];
print(strArr[0]);

int fib(int x)
{
  if (x < 2) return 1;
  return fib(x-1) + fib(x-2);
}
print(fib(0));
print(fib(3));