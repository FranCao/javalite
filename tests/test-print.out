12
-6
0
Foo
0.423313
0
0
1
hello world
11
hello
1
3